tests/Integration/inc/Engine/CDN/Subscriber/TestCase.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\Engine\CDN\Subscriber;
+
+use WPMedia\PHPUnit\Integration\TestCase as BaseTestCase;
+
+class TestCase extends BaseTestCase {
```

Comment:
Make this `abstract`.

Reply:
https://github.com/wp-media/wp-rocket/pull/2589/commits/680c402d2ee1b0611609890244583b4b81a76e49